setup.cfg

```diff
@@ -0,0 +1,68 @@
+[metadata]
+name = octodns
+description = "DNS as code - Tools for managing DNS across multiple providers"
+long_description = file: README.md
+version = attr: octodns.__VERSION__
```

Comment:
Cool. This seems like a good option.

Reply:
Yep! No code, only retrieve at build/install and on a specified file, it's more an elegant way.
You're welcome!